docs/internals/contributing/index.txt

```diff
@@ -65,7 +65,7 @@ Django community and others to maintain a great ecosystem to work in:
   friendly and helpful atmosphere. If you're new to the Django community,
   you should read the `posting guidelines`_.
 
-* Join the `#django IRC channel`_ on Libera.Chat and answer questions. By
+* Join the `Django Discord Server`_ or the `#django IRC channel`_ on Libera, to chat and answer questions. By
```

Comment:
`Libera.Chat` is a name. Also, wrap at 79 chars: 
```
* Join the `Django Discord Server`_ or the `#django IRC channel`_ on
  Libera.Chat and answer questions. By explaining Django to other users, you're
  going to learn a lot about the framework yourself.
```

Reply:
ohh my bad didn't know about it, let me push new changes